### PR TITLE
Feature/seab 314/cd

### PR DESCRIPTION
**Pull request by coverbeck**
* Change upload to s3 to also upload to /develop folder
* Then invalidate the CloudFront distribution.

Notes:

1. Will change feature branch to develop in config.yml when tests pass.
2. Tests not passing because Circle CI user does not have IAM permissions to invalidate CF.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -100,11 +101,44 @@ jobs:
...
+                  --exclude index.html
+            - aws-s3/copy:
+                from: dist/index.html
+                to: 's3://${AWS_BUCKET}/develop/'
+                arguments: |
+                  --cache-control max-age=0
+  update_cloudfront:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/python:2.7
```
Comment (denis-yuen) [inline .circleci/config.yml]: hmmm, rats
https://github.com/aws/aws-cli/issues/3553

**Comment by denis-yuen** [comment]
Well, looks like the circle ci update_cloudfront thinks it passed now

File: .circleci/config.yml [hunk 2]
```diff
@@ -100,11 +101,44 @@ jobs:
...
                 arguments: |
                   --cache-control max-age=0
+            # Copy to a "live" develop folder
+            - aws-s3/copy:
+                from: dist
+                to: 's3://${AWS_BUCKET}/develop/'
+                arguments: |
+                  --recursive \
+                  --exclude index.html
+            - aws-s3/copy:
```
[the remark this answers is not in the document]
Comment (coverbeck) [reply .circleci/config.yml]: A new index.html will reference new JavaScript files, so it won't technically invalidate the JS caching, but render it moot, as the older version files won't be used any more.

File: .circleci/config.yml (hunk 1 above)
Comment (coverbeck) [reply .circleci/config.yml]: There's a [Circle CI orb](https://circleci.com/orbs/registry/orb/circleci/aws-cli) for AWS-CLI, but it has no examples of how to invoke a command. So I gave up and went this route. :(

**Pull request merged**